### DISCUSSION TITLE
Parquet exporter status

### DISCUSSION THREAD

**Discussion by punya** [open-telemetry/opentelemetry-collector-contrib#27072] [General]
The [Parquet exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/parquetexporter) was introduced in 2021 but development seems to have stalled out after creating the initial shim in [#6903](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/6903). Is there any plan to continue working on this, or is the intention to support this need through ongoing work on OTel Arrow?

cc @atoulme @jmacd.

**Comment by crobert-1** [comment]
Apologies for the delay @punya, it looks like the decision was made to remove it in #27284.

**Reply by atoulme** [reply]
Indeed, Otel Arrow seems like a very promising approach, you should concentrate your efforts there.
